src/components/waste/WasteInputForm.tsx

```diff
@@ -84,7 +85,7 @@ export const WasteInputForm = ({
       (inputValueCitySelected ? filteredStreets : filteredCities)?.length < 6
         ? device.platform === 'ios'
           ? 'auto'
-          : (inputValueCitySelected ? filteredStreets : filteredCities)?.length *
+          : (inputValueCitySelected ? filteredCities : filteredStreets)?.length *
```

Comment:
i do not understand this change for height calculation. can you please explain the swap

Reply:
If `inputValueCitySelected` is true then the `filteredStreet.length` is 0 because the city is selected but the `filteredCities.length` is greater than 0. Therefore the list of suggestions for Android was not visible. So I changed `filteredScreets` to `filteredCities`.

Reply:
hm, I do not get it. if the city is selected, we want to show streets next, so we need to calculate with street length.
if a city is not selected, we want to show cities and calculate with cities. this is how can think about the situation.

otherwise the line before `(inputValueCitySelected ? filteredStreets : filteredCities)?.length < 6` needs an update as well?

Reply:
Yeah, you're right. That doesn't make sense right now. How about updating the code in this way to make it more understandable? We're currently doing the same check in 2 different places. I haven't tested this code yet.

```
 const listContainerHeight = useMemo(() => {
    const activeList = inputValueCitySelected ? filteredStreets : filteredCities;
    const listLength = activeList?.length ?? 0;

    if (listLength < 6) {
      return device.platform === 'ios' ? 'auto' : listLength * (normalize(22) + 2 * normalize(16));
    } else {
      return dimensions.height / 2.5;
    }
  }, [inputValueCitySelected, filteredStreets, filteredCities, dimensions.height]);
```